Src/Fido2/AuthenticatorResponse.cs

```diff
@@ -68,10 +68,19 @@ protected void BaseVerify(string expectedOrigin, byte[] originalChallenge, byte[
                 throw new Fido2VerificationException("Challenge not equal to original challenge");
 
             // 5. Verify that the value of C.origin matches the Relying Party's origin.
-            if (Origin != expectedOrigin)
-                throw new Fido2VerificationException($"Origin {Origin} not equal to original origin {expectedOrigin}");
+            if (expectedOrigins == null || expectedOrigins.Length < 1 || expectedOrigins[0] == null)
```

Comment:
nit: When checking if an array is empty, it feels more natural to use `== 0`. It isn't like it's going to contain a negative amount of elements.

Reply:
Not sure if I see the big difference, but I updated the code.

Reply:
The difference is in intention; the former gives the impression that the value actually _can_ be less than 0.